src/test/java/com/springbootpractice/examples/employeemanagement/integrationtest/EmployeeBDD.java

```diff
@@ -0,0 +1,57 @@
+package com.springbootpractice.examples.employeemanagement.integrationtest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.springframework.web.client.RestTemplate;
+
+import com.springbootpractice.examples.employeemanagement.data.Employee;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class EmployeeBDD extends SpringIntegrationTest {
+
+	private final RestTemplate restTemplate = new RestTemplate();
+
+	private Employee employee = new Employee();
+
+	private Employee emp;
+
+	@Given("a valid employee object")
```

Comment:
Where is the FF for this line ? and few lines in the stepDef am not able to find valid step in FF